README.md

```diff
@@ -118,7 +118,7 @@ var kvps = DotNetEnv.Env.Load(
 )
 
 // or the recommended, cleaner (fluent) approach:
-var dict = DotNetEnv.Env.NoEnvVars().Load().ToDictionary();
+var dict = DotNetEnv.Env.NoEnvVars().Load().ToDictionary(CreateDictionaryOption.TakeLast);
```

Comment:
I would want this example to not change.

Reply:
Outdated due to renaming the ExtensionMethod, further discussion see there: https://github.com/tonerdo/dotnet-env/pull/88#discussion_r1431622773